src/app/components/chat/bubbles/bubbles-directive.js

```diff
@@ -41,21 +41,9 @@ angular.module( 'App.Chat' ).directive( 'gjChatTabs', function( Screen, Chat, Sh
 				Chat.client.leaveRoom( roomId );
 			};
 
-			$scope.$watch( 'Chat.client.hasOpenRooms() && !Screen.isXs', function( isShowing )
+			$scope.$watch( '!Screen.isXs', function( isShowing )
```

Comment:
This is silly. In stylus just put a breakpoint in the styles that effectively do what chat-tabs-visible does?

```stylus
display: none
@media $media-sm-up
  display: block
```